phoebe/atmospheres/passbands.py

```diff
@@ -1631,46 +1363,46 @@ def _log10_Inorm(self, query_pts, atm, intens_weighting='photon', atm_extrapolat
 
         if blending_method == 'blackbody':
             raise NotImplementedError('under review.')
-            log10_Inorm_bb = np.log10(self.Inorm(atm='blackbody', query_pts=query_pts[nanmask], ldatm=atm, ld_extrapolation_method=ld_extrapolation_method, intens_weighting=intens_weighting))
-            nv, naxes = ndpolator.map_to_cube(req[nanmask], self.atm_axes[atm][:-1], self.blending_region[atm], return_naxes=True)
+            # log10_Inorm_bb = np.log10(self.Inorm(atm='blackbody', query_pts=query_pts[nanmask], ldatm=atm, ld_extrapolation_method=ld_extrapolation_method, intens_weighting=intens_weighting))
+            # nv, naxes = ndpolator.map_to_cube(req[nanmask], self.atm_axes[atm][:-1], self.blending_region[atm], return_naxes=True)
 
-            log10_Inorm_bl = np.empty_like(teffs[nanmask])
-            for si, selem in enumerate(nv):
-                ic = [np.searchsorted(naxes[k], selem[k])-1 for k in range(len(naxes))]
-                seps = (np.abs(ndp.ics[atm]-np.array(ic))).sum(axis=1)
-                corners = np.argwhere(seps == seps.min()).flatten()
+            # log10_Inorm_bl = np.empty_like(teffs[nanmask])
+            # for si, selem in enumerate(nv):
+            #     ic = [np.searchsorted(naxes[k], selem[k])-1 for k in range(len(naxes))]
+            #     seps = (np.abs(ndp.ics[atm]-np.array(ic))).sum(axis=1)
+            #     corners = np.argwhere(seps == seps.min()).flatten()
```

Comment:
(and all following sections) can we just delete these instead of commenting them out?  They'll always be in the git history if we need to find them to add them back in?

Reply:
I am tackling this next and it's useful to have it in front of me rather than fishing through git history. I'll remove it in my next (related) PR.